### ISSUE TITLE
Is there some way to download the images for Celeba-HQ

### ISSUE THREAD

**Issue by pjreddie** [nashory/pggan-pytorch#21]
Or if you can make them available to me I can host them somewhere for people to download. I'd really like the dataset but don't particularly want to go through the process to generate it myself.

**Comment by nashory** [comment]
The authors released the dataset, and here is the link 
https://drive.google.com/open?id=0B4qLcYyJmiz0TXY1NG02bzZVRGs

please refer to this repo .
https://github.com/tkarras/progressive_growing_of_gans

**Comment by petergerten** [comment]
@nashory seems it is only the pretrained model, not the source images

**Comment by darthdeus** [comment]
@petergerten aren't they stored in the .dat files in the `datasets` directory? https://drive.google.com/drive/folders/1iGK3jDe_1-V0Lyy5468WNpb7y74u-iEp

**Comment by nperraud** [comment]
The `.dat` files from google drive only contain the difference between the celebA and the celebA-HQ images. The process to create the images is actually quite complicated. 
I just made my scripts  available at 
https://github.com/nperraud/download-celebA-HQ
They are far from perfect but I hope it helps.

**Comment by BKHMSI** [comment]
Hi @nperraud! Thanks for sharing the scripts. I am trying to download the deltas from Google Drive but it gives me an error that the "download quota exceeded for this file" for all files of course. I guess there is a limit to the number of downloads per day for Google Drive. Do you have any idea if I can find the files stored somewhere else?

**Comment by suvojit-0x55aa** [comment]
I generated the files and stored it as convenient zip files. You can download from Google Drive or generate using pre-built docker image. Visit [suvojit-0x55aa/celebA-HQ-dataset-download](https://github.com/suvojit-0x55aa/celebA-HQ-dataset-download)

**Comment by shartoo** [comment]
@suvojit-0x55aa  there should be 200k images in Celeba HQ,but your sharing are 30k of them.

**Comment by suvojit-0x55aa** [comment]
@shartoo As quoted in the Progressive Growing of GANs [paper](https://research.nvidia.com/sites/default/files/pubs/2017-10_Progressive-Growing-of/karras2018iclr-paper.pdf)

> In this section we describe the process we used to create the high-quality version of the CELEBA dataset, consisting of 30000 images in 1024 × 1024 resolution.

The Celeba HQ dataset contains only 30k images.

**Comment by shartoo** [comment]
sorry for  my careless check,you're right.

**Comment by shartoo** [comment]
Another question,how could i use the `list_attr_celaba.txt` namely attribution label for celaba-hq.Does the 30k images of celeba-hq are first 30k of 200k images in celaba dataset?

**Comment by shartoo** [comment]
Got it ,there is a image list file https://raw.githubusercontent.com/nperraud/download-celebA-HQ/master/image_list.txt which releated celaba and celaba-hq files.

**Comment by Cuilie** [comment]
> @shartoo As quoted in the Progressive Growing of GANs [paper](https://research.nvidia.com/sites/default/files/pubs/2017-10_Progressive-Growing-of/karras2018iclr-paper.pdf)
> 
> > In this section we describe the process we used to create the high-quality version of the CELEBA dataset, consisting of 30000 images in 1024 × 1024 resolution.
> 
> The Celeba HQ dataset contains only 30k images.

Is It possible to share labels with the dataset? Thanks!

**Comment by suvojit-0x55aa** [comment]
@Cuilie you can use [this file](https://raw.githubusercontent.com/nperraud/download-celebA-HQ/master/image_list.txt).

**Comment by aciobanusebi** [comment]
@suvojit-0x55aa On your github page you said "The size of the final dataset is 89G". But in Google Drive, you have far less than 89Gb. How come?

**Comment by suvojit-0x55aa** [comment]
@aciobanusebi as far as I remember that's the memory needed for raw dataset, the deltas, and the generated dataset.
